docs/source/tutorials/ontologies.rst

```diff
@@ -0,0 +1,26 @@
+********
+Ontologies
+********
+
+A central component of a project is the ontology - an in-depth description of which can be found `here <https://docs.encord.com/ontologies/overview>`_.
+
+The ontology essentially defines the label structure of a given project.
+
+- For a platform description of the ontology, please see :xref:`configure_label_editor_(ontology)`.
+- To access the ontology you can use the :meth:`~encord.user_client.EncordUserClient.get_ontology()` method.
+    - This allows you to work with its `structure` property to read or write parts of the ontology.
+- The structure comes as the :class:`~encord.ontology.ontology_labels_impl.OntologyStructure` class which has good self documenting examples on how to be used.
+
+This section will cover some of the most common use cases for ontologies.
+
+Copying an ontology
+==================
+
+An Ontology can be copied to be used in another project without the need to re-create all objects or classifications from scratch. 
```

Comment:
I think this point is misleading. The reason why ontologies are separate entities from projects is because we wanted to use the same ontology in multiple projects. 
The reason why you would want to copy an ontology is because you want to have a new ontology attached to a different project, which means that you could change the editor/structure of this ontology without affecting other projects. 
It is very important to make this distinction.